### ISSUE TITLE
Hilar lymph node enlargement

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#7701]
**Preferred term label:** 
Hilar lymph node enlargement

**Synonyms**
Hilar lymphadenopathy

**Definition (free text, please give PubMed ID)**
Hilar lymphadenopathy is swelling or enlargement of lymph nodes localised in hila, wedge-shaped anatomical structures containing pulmonary vessels, major bronchi, nerves and lymph nodes. Hilar lymph nodes are not visible on chest X-ray if they are not enlarged. Hilar lymph node enlargement can be caused by benign and malignant conditions and can be symmetrical (bilateral) or asymmetrical (unilateral).

**Parent term (use hpo.jax.org/app)**
HP:0002716

**Diseases characterized by this term? (e.g. Orphanet or OMIM number)**
sarcoidosis
tuberculosis
histoplasmosis
coccidioidomycosis
malignancy
silicosis
berylliosis
heart failure

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
added as HP:0034388
